aws_marketplace/using_model_packages/preprocessing-audio-data-using-a-machine-learning-model/preprocessing-audio-data-using-a-machine-learning-model.ipynb

```diff
@@ -0,0 +1,894 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    !rm -rf $RAW_DATA
</pre><p>I don't recommend this hard recursive delete of anything in a folder called "data". That could easily already be in use for other purposes and could delete a customer's important data. You could use the default <code>/tmp/</code> folder, or something more descriptive like <code>DEMO-midii-audio/</code>	.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3061/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>